test/java/org/opendatakit/briefcase/export/CsvTest.java

```diff
@@ -84,9 +84,26 @@ public void includes_non_repeat_groups_in_repeat_filenames2() {
     assertThat(exportDir.resolve("some_form-r3.csv"), exists());
   }
 
+  @Test
+  public void sanitizes_filenames() {
+    Model group = new ModelBuilder()
+        .addGroup("data")
+        .addGroup("some-group")
+        .addRepeatGroup("re\tpeat")
+        .build();
+
+    FormDefinition formDef = buildFormDef("some.,form", group, 2);
+
+    Csv.main(formDef, conf).prepareOutputFiles();
+    Csv.repeat(formDef, group, conf).prepareOutputFiles();
+
+    assertThat(exportDir.resolve("some__form.csv"), exists());
+    assertThat(exportDir.resolve("some__form-re peat.csv"), exists());
+  }
+
   private ExportConfiguration buildConf(Path exportDir) {
     return new ExportConfiguration(
-        Optional.of("some_form.csv"),
+        Optional.empty(),
```

Comment:
Setting the filename to `some_form.csv` here could be misleading since we want to test that we compose correctly a filename using the form's name and the repeat name. 

That's why I've replaced that with `Optional.empty()`